### PR TITLE
add new blog post

### PR DESCRIPTION
**Pull request by aaronpk**

### CODE REVIEW

File: _source/_posts/2018-08-14-php-crud-app-symfony-angular.md [hunk 1]
```diff
@@ -0,0 +1,1069 @@
...
+
+So that covers fast and dynamic, but what about secure? For that you''ll use Okta for user authentication and authorization. You'll need to set up a development environment with PHP 7 and Node.js 8+/npm. You will also need an [Okta developer account](https://developer.okta.com/).
+
+## Why Okta?
+
+Well, we might be biased, but we think Okta makes [identity management](https://developer.okta.com/product/user-management/) easier, more secure, and more scalable than what you're used to. Okta is an API service that allows you to create, edit, and securely store user accounts and user account data, and connect them with one or more applications. Our API enables you to:
+
+* [Authenticate](https://developer.okta.com/product/authentication/) and [authorize](https://developer.okta.com/product/authorization/) your users
+* Store data about your users
+* Perform password-based and [social login](https://developer.okta.com/authentication-guide/social-login/)
```
Comment (jmelberg-okta) [inline _source/_posts/2018-08-14-php-crud-app-symfony-angular.md]: For internal links (ones on this repo), these should be relative:
`/authentication-guide/social-login/`

File: _source/_posts/2018-08-14-php-crud-app-symfony-angular.md [hunk 2]
```diff
@@ -0,0 +1,1069 @@
...
+
+    // Attempt authorization with the provided token
+    try {
+
+        // Setup the JWT Verifier
+        $jwtVerifier = (new \Okta\JwtVerifier\JwtVerifierBuilder())
+                        ->setAdaptor(new \Okta\JwtVerifier\Adaptors\SpomkyLabsJose())
+                        ->setAudience('api://default')
+                        ->setClientId('{yourClientId}')
+                        ->setIssuer('https://{yourOktaDomain}.com/oauth2/default')
```
Comment (jmelberg-okta) [inline _source/_posts/2018-08-14-php-crud-app-symfony-angular.md]: `https://{yourOktaDomain}.com` -> `https://{yourOktaDomain}`

This is why the Travis tests are failing.

File: _source/_posts/2018-08-14-php-crud-app-symfony-angular.md (hunk 1 above)
Comment (aaronpk) [reply _source/_posts/2018-08-14-php-crud-app-symfony-angular.md]: Thanks, I keep forgetting to fix these links at the end. I thought blog links were supposed to be relative but ones in the rest of the site were supposed to be absolute?

Comment (jmelberg-okta) [reply _source/_posts/2018-08-14-php-crud-app-symfony-angular.md]: Close! Anything that isn't served from `okta.github.io` needs to be absolute. The common ones are:
- `/`
- `/product/`
- `/signup/`

Everything else can be relative.

**Pull request merged**